etc/ReplayOfflineConfiguration.py

```diff
@@ -123,9 +125,9 @@
 alcarawProcVersion = dt
 
 # Defaults for GlobalTag
-expressGlobalTag = "120X_dataRun3_Express_v2"
-promptrecoGlobalTag = "120X_dataRun3_Prompt_v2"
-alcap0GlobalTag = "120X_dataRun3_Prompt_v2"
+expressGlobalTag = "121X_dataRun3_Express_Candidate_2021_11_18_19_28_59"
+promptrecoGlobalTag = "121X_dataRun3_Prompt_Candidate_2021_11_18_19_44_08"
+alcap0GlobalTag = "121X_dataRun3_Prompt_Candidate_2021_11_18_19_44_08"
```

Comment:
```
$ conddb list 121X_dataRun3_Prompt_Candidate_2021_11_18_19_44_08 | grep EcalLaserAPDPNRatiosRcd
$ conddb list EcalLaserAPDPNRatios_prompt_v3 | head -10
[2021-11-19 10:33:37,938] INFO: Connecting to pro [frontier://PromptProd/cms_conditions]
[2021-11-19 10:33:38,089] INFO: Listing with a limit of 500 IOVs, starting from the highest since. If you need to see more, please increase the limit of returned results.
Since: UTC          (timestamp)            Insertion Time       Payload                                   Object Type            
-----------------------------------------  -------------------  ----------------------------------------  --------------------   
2021-10-20 06:53:37 (7021038087367032832)  2021-10-20 08:07:33  3608e06699aa76bada9b330d8bad4976d2b92558  EcalLaserAPDPNRatios   
2021-10-20 07:19:17 (7021044701616668672)  2021-10-20 08:44:18  aedf4685ef2c0330b587475825e06c6f1a75e0b7  EcalLaserAPDPNRatios   
2021-10-20 07:39:09 (7021049821217685504)  2021-10-20 13:56:12  98ff1a8f3056ad9d508d1eeb5772bc22e883b42a  EcalLaserAPDPNRatios   
2021-10-20 08:23:40 (7021061293075333120)  2021-10-20 14:45:45  6e282a4da4529d0bb6fc6b7133ea31c9c42d40d0  EcalLaserAPDPNRatios   
2021-10-20 13:35:15 (7021141587488931840)  2021-10-20 15:23:11  793a6458f3c1d25d2a8ceb1a4211ea8bb149cbe2  EcalLaserAPDPNRatios   
2021-10-20 14:19:09 (7021152900432789504)  2021-10-20 17:15:46  3136beb8de3d70a8a4f0e815dccff04f6d987a66  EcalLaserAPDPNRatios   
2021-10-20 15:03:03 (7021164213376647168)  2021-10-20 18:30:29  9d7fa409db8289d5916b7adf6adf7c988ffdf12d  EcalLaserAPDPNRatios   
2021-10-20 16:53:13 (7021192603110473728)  2021-10-20 19:08:00  a7726a915afbbc9df90c9f2f5a05cdfec7a215b9  EcalLaserAPDPNRatios  
```

there's no IOV in this record covering for 2018 data ... 
That's the sort of annoyances that I tried to warn about @tvami @francescobrivio and @malbouis here: https://github.com/cms-sw/cmssw/pull/35593#issuecomment-940121649